src/app/views/dashboard/account/linked-accounts/linked-accounts.vue

```diff
@@ -136,7 +139,7 @@ export default class RouteDashAccountLinkedAccounts extends LegacyRouteComponent
 				<AppLinkedAccount
 					:account="facebookAccount"
 					:disabled="loading"
-					provider="facebook"
+					provider="accountProviderFacebook"
```

Comment:
a bit confusing, but in order to bind this as the value and not a string, you have to put a `:` before it, similar to acocunt and disabled. It's why the syntax color of the value is different in the IDE.

Same with the below changes